Data-Structure/Collection-In-Java/CollectionOfEmployee/Employee.java

```diff
@@ -0,0 +1,71 @@
+package CollectionOfEmployee;
+
+import java.util.Comparator;
+
+/**
+ * @author Niraj Gupta
+ * Class to represent an Employee with id, name and address.
+ * It also implements the comparable interface for sorting employee.
+ */
+public class Employee implements Comparable<Employee>{
+	private String name;
+	private String id;
+	private String address;
+
+	/**
+	 * Parameterized Constructor
+	 * @param id : The unique id given to an employee.
+	 * @param name : The name of the employee.
+	 * @param address : The address of the employee.
+	 */
+	public Employee(String id, String name, String address){
+		this.id = id;
+		this.name = name;
+		this.address = address;
+	}
+
+	public String getName(){
+		return name;
+	}
+
+	public String getId(){
+		return id;
+	}
+
+	public String getAddress(){
+		return address;
+	}
+
+	/**
+	 * Method to Compare Employee based in their ID.
+	 */
+	@Override
+	public int compareTo(Employee employee){
+		return id.compareTo(employee.getId());
+	}
+
+	/**
+	 * Comparator class to sort employee based on their name.
+	 */
+	public static Comparator<Employee> NameComparator = new Comparator<Employee>(){

```

Comment:
we do not use method like specifications for variables, if needed you could have added a inline comment starting with //

Reply:
ok.. from next time i will incorporate the above suggestion.